src/main/java/seedu/duke/helper/command/Command.java

```diff
@@ -12,7 +12,7 @@
 
 
 public abstract class  Command {
-    protected String[] parameterArray;
+    public String[] parameterArray;
 
     Command(String[] parameterArray) {
         this.parameterArray = parameterArray;
```

Comment:
Replace all the // lines with 
/*
all the codes
*/
so that it could pass the check style before we merge